core/health/api/src/main/java/org/opennms/core/health/api/HealthCheck.java

```diff
@@ -44,6 +47,16 @@
      */
     String getDescription();
 
+    /**
+     *
+     * A tag to indicate the category of the health-check.
+     *
+     * @return The HealthTag type, describing category of the health-check.
+     */
+    default List<String> getTags(){
```

Comment:
We can force implementations to adopt this. No need for default

Reply:
Okay, either way works for me, all depend on how we want to design this.  Changed.